Packages/Tracking/Examples~/Hands Examples/Scripts/CycleHandPairs.cs

```diff
@@ -13,40 +13,47 @@ namespace Leap.Unity.HandsModule.Examples
 {
     public class CycleHandPairs : MonoBehaviour
     {
-        public List<GameObject> handList;
+        [SerializeField]
+        private HandModelManager handModelManager;
         private int currentHandID;
 
         // Use this for initialization
         void Start()
         {
+            if (handModelManager == null)
+            {
+                handModelManager = FindObjectOfType<HandModelManager>();
+            }
             currentHandID = 0;
+            handModelManager.EnableHandModelPair(currentHandID, true);
```

Comment:
Check for null?

Reply:
done! was worth doing as someone might reuse this code.